solutions/cf-1418C.mdx

```diff
@@ -2,27 +2,27 @@
 id: cf-1418C
 source: CF
 title: Mortal Kombat Tower
-author: Ryan Chou
+author: Ryan Chou, Chuyang Wang
 ---
 [Official Editorial (C++)](https://codeforces.com/blog/entry/82673)
 
 ## Alternate Solution - DP
 ### Explanation
 
-If we define $\texttt{dp}[i][j]$ as the minimum number of skip points needed to get to the $i$th boss on turn $j$ 
+If we define $\texttt{dp}[i][j]$ as the minimum number of skip points needed to get to the $j$th boss on player $i$'s turn 
```

Comment:
why swap the two?

Reply:
cause it was the other way around in the code. I either have to write $\texttt{dp}[j][i]$ or swap the definition of those two vars, or, change the cpp code completely to make it a dp[n][2] array (it was dp[2][n]).

Reply:
fair enough